DXRMissions/DeusEx/Classes/DXRMissionsM01.uc

```diff
@@ -108,6 +108,7 @@ function AddMissionGoals()
 {
     local DeusExGoal newGoal;
     if(dxr.localURL != "01_NYC_UNATCOISLAND") return;
+    if(dxr.flags.IsReducedRando()) return;
```

Comment:
Maybe the parent shouldn't be calling AddMissionGoals() at all in reduced Rando modes?